.gitignore

```diff
@@ -1 +1,4 @@
-/target
\ No newline at end of file
+/target
+spdx-sbom-generator*
```

Comment:
I think you can remove this line, because the binary is going to be downloaded under `/bin`, which is already ignored